packages/frontend/src/components/ExploreTree.tsx

```diff
@@ -0,0 +1,176 @@
+import {FC, useCallback, useMemo, useState} from "react";
```

Comment:
So no problems here with state not updating in the blueprint tree?

Reply:
Haven't noticed any problems. Things I tried:
- expanding/collapsing trees ✅ 
- selecting/unselecting metrics/dimensions ✅ 
- go back to tables and explore a different table ✅ 
- change dimensions in dbt and refresh 👎  ( this doesn't update the UI because we aren't making a new call to get the refreshed table )

Is there something else to test ?

Reply:
That's great!